src/pages/workspaces/workspace/Data.js

```diff
@@ -520,9 +520,8 @@ const ToolDrawer = _.flow(
               onClick: () => setToolMode('Notebook'),
               disabled: !notebookButtonEnabled,
               tooltip: Utils.cond(
-                [!entityMetadata.cohort, () => 'Talk to your dataset owner about setting up a Data Explorer. See the "Making custom cohorts with Data Explorer" help article.'],
                 [isCohort && entitiesCount > 1, () => 'Select exactly one cohort to open in notebook'],
-                [!isCohort, () => 'Only cohorts can be opened with notebooks'],
+                [!isCohort || !entityMetadata.cohort, () => 'Only cohorts can be opened with notebooks'],
```

Comment:
nit: I don't think the change to this line does anything, because having a cohort selected implies that cohorts exist as a type in the metadata.